doc/source/ray-core/handling-dependencies.rst

```diff
@@ -403,8 +403,6 @@ Runtime environment resources on each node (such as conda environments, pip pack
 
 When the cache size limit is exceeded, resources not currently used by any actor, task or job will be deleted.
 
```

Comment:
Since this is always true, should we make this in present tense.

That is, "When the cache size limit is exceeded, resources not currently used by any actor, task or job are deleted."
"When" is conditional like if, so future tense works tool. I like to keep things that are true or factual in present tense. 

So either will work , so long as we are consistent.

Reply:
Thanks makes sense! I'll need to do a full pass over the runtime env doc checking for things like this